src/viam/sdk/tests/mocks/mock_robot.cpp

```diff
@@ -1,9 +1,13 @@
-#include <vector>
 #include <viam/sdk/tests/mocks/mock_robot.hpp>
 
+#include <vector>
+
+#include <grpcpp/support/status.h>
+
 #include <common/v1/common.pb.h>
 #include <robot/v1/robot.pb.h>
 
+#include <src/viam/sdk/common/private/version_metadata.hpp>
```

Comment:
I cannot figure out why this has to be `#include <*src*/viam/sdk...>` when every other SDK include is `#include <viam/sdk/...>`. I tried messing around with `include_directories` in the test `CmakeLists.txt`, but couldn't figure this out and struggled to find any guidance online. If anyone has thoughts here I'd appreciate them!